service/routers/api.py

```diff
@@ -28,10 +34,23 @@ async def recognize(
     # Calculate the duration of the audio
     duration = librosa.get_duration(y=y, sr=sr)
 
+    # Create a torch.Tensor from the audio data
+    audio_tensor = load_audio(io.BytesIO(wav_data), sr=sr)
+
+    # Noise reduction
+    audio_tensor = denoise_model.recognize(audio_tensor)
+    # Save the audio to temporary file
+    with tempfile.NamedTemporaryFile(delete=False, suffix='.waw') as tmp_file:
```

Comment:
Can we avoid writing to a temporary file? Can the model work directly with bytes?

Reply:
the model can't work directly with bytes